src/System.IO.UnmanagedMemoryStream/tests/UmsWrite.cs

```diff
@@ -109,18 +109,10 @@ public static void CannotWriteWithOverflow()
                 UnmanagedMemoryStream stream = manager.Stream;
                 UmsTests.WriteUmsInvariants(stream);
 
-                if (IntPtr.Size == 4)
-                {
-                    Assert.Throws<ArgumentOutOfRangeException>(() => stream.Position = long.MaxValue);
-                    stream.Position = int.MaxValue;
-                }
-                else
-                {
-                    stream.Position = long.MaxValue;
-                    var bytes = new byte[3];
-                    Assert.Throws<IOException>(() => stream.Write(bytes, 0, bytes.Length));
-                    Assert.Throws<IOException>(() => stream.WriteByte(1));
-                }
+                stream.Position = long.MaxValue;
```

Comment:
This was fixed here:
https://github.com/dotnet/corefx/pull/19440/commits/4fc34e40dc5f570bc8dfd6b77a49b0508c8cfd7e#diff-ca9df2124bf0eb4c3a89070bbd960b5aR112